### ISSUE TITLE
Installation issues: Vispy and Conda?

### ISSUE THREAD

**Issue by sanjayankur31** [NeuroML/pyNeuroML#410]
Our 3D vispy based viewer doesn't work out of the box for people that use conda to create their virtual environments. There seems to be some sort of conflict between the conda packages and pip packages. We need to investigate this and see if there's a way to work around this.

Perhaps we also need to add a GHA to test some stuff with a conda virtual env

**Comment by sanjayankur31** [comment]
I really can't get it to work on conda at all. We need to see if vispy examples work in conda. If they don't, perhaps we can detect conda usage and print out a warning to users asking them to use non-conda virtual environments.

**Comment by sanjayankur31** [comment]
So, I installed conda and tried to do a fresh install of vispy etc. It didn't work. With 3.11, the first example I ran just sat there. With 3.9, it crashed because numpy's c-extensions could not be loaded.

So, as far as we're concerned, pyneuroml *should* work with conda, but other extras such as vispy may or may not work for different people. We recommend they use "normal" python environments where we know stuff works. If they want to use conda, they'll have to figure some stuff out themselves, unfortunately---it's not quite possible for us to test all our dependencies on conda.
